project_one.html

```diff
@@ -222,11 +223,13 @@ <h2>Contact me</h2>
             />
             <input
               type="email"
+              class="contactme"
               name="user_email"
               id="email"
               placeholder="Enter your email"
               required
             />
+            <div class="validate" > Email must in Lower case </div>
```

Comment:
- [OPTIONAL] I suggest you to wrap the text in a p tag following semantic HTML and kindly fix the misspelling. Keep coding 💪